src/main/java/org/wso2/carbon/inbound/poll/dbeventlistener/DBEventPollingConsumer.java

```diff
@@ -0,0 +1,380 @@
+/*
+* Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.wso2.carbon.inbound.poll.dbeventlistener;
+
+import org.apache.axiom.om.OMAbstractFactory;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.OMFactory;
+import org.apache.axiom.om.util.UUIDGenerator;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.core.SynapseEnvironment;
+import org.apache.synapse.mediators.base.SequenceMediator;
+import org.wso2.carbon.base.MultitenantConstants;
+import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.inbound.endpoint.protocol.generic.GenericPollingConsumer;
+
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Properties;
+
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.Statement;
+import java.sql.DriverManager;
+import java.sql.Types;
+
+public class DBEventPollingConsumer extends GenericPollingConsumer {
+
+    private static final Log log = LogFactory.getLog(DBEventPollingConsumer.class);
+    
+    private String driverClass;
+    private String dbURL;
+    private String dbUsername;
+    private String dbPassword;
+    private String filteringColumnName;
+    private String filteringCriteria;
+    private Connection connection = null;
+    private MessageContext msgCtx;
+    private String deleteQuery = null;
```

Comment:
Not required to have those as class level variables. Please check and move to method level.